src/EnvUtils.h

```diff
@@ -22,8 +22,6 @@ QString getenv(QString name);
 
 bool setenv(QString name, QString value);
 
-bool waylandIsUsed();
```

Comment:
Why remove that function? It might be handy to keep around for future uses. And if it is unused, it won't get linked into the final executable anyway.